app/both/_config/adminConfig.js

```diff
@@ -0,0 +1,123 @@
+AdminConfig = {
+  name: 'Waartaa',
+  adminEmails: ['sheeshmohsin@gmail.com'],
```

Comment:
We should not hard code such values. We should pick it up from ``Meteor.settings``